### PR TITLE
(GH-322) Support deleting comments

### LINKED ISSUE #322: Support for deleting comments
Please provide an option to delete pull request thread comments.

### PR DESCRIPTION
**Pull request by twenzel**
Provides new methods on `AzureDevOpsPullRequest` to delete, update and create comments
Fixes #322

### CODE REVIEW

File: src/Cake.AzureDevOps/Repos/PullRequest/CommentThread/AzureDevOpsPullRequestCommentThread.cs [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 {
-    using System;
-    using System.Collections.Generic;
-    using System.Linq;
     using Cake.Core.IO;
     using Microsoft.TeamFoundation.SourceControl.WebApi;
     using Microsoft.VisualStudio.Services.WebApi;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
```
Comment (pascalberger) [inline src/Cake.AzureDevOps/Repos/PullRequest/CommentThread/AzureDevOpsPullRequestCommentThread.cs:8]: Revert reordering of usings. We have configured linting rule for `System` namespaces to appear before other usings.

**Review by pascalberger — CHANGES_REQUESTED** [review]
Just one small comment. Otherwise LGTM

File: src/Cake.AzureDevOps/Repos/PullRequest/CommentThread/AzureDevOpsPullRequestCommentThread.cs (hunk 1 above)
Comment (twenzel) [reply src/Cake.AzureDevOps/Repos/PullRequest/CommentThread/AzureDevOpsPullRequestCommentThread.cs]: Done.

**Review by pascalberger — APPROVED** [review]
LGTM

**Comment by pascalberger** [comment]
@twenzel your changes have been merged, thanks for your contribution 👍

**Pull request merged**